compiler/tensor.cc

```diff
@@ -83,16 +86,13 @@ void DumpDataToRepeated(const Tensor& t, ::google::protobuf::RepeatedField<To>*
     DumpDataToRepeated<To, To>(t, a);
 }
 
-}  // namespace
-
-Tensor::Tensor(const onnx::TensorProto& xtensor)
-    : dims_(xtensor.dims().begin(), xtensor.dims().end()),
-      dtype_(xtensor.data_type()),
-      data_(nullptr, &std::free),
-      name_(xtensor.name()),
-      doc_string_(xtensor.doc_string()) {
+chainerx::Array TensorProtoToArray(onnx::TensorProto const& xtensor) {
```

Comment:
Please revive initializations for `name_` and `doc_string_`. My bad this was not caught by a test...

Reply:
This just splits the creation of `chainerx::Array` so constructor is still there.